opencti-platform/opencti-graphql/tests/utils/testQuery.ts

```diff
@@ -80,3 +80,12 @@ export const serverFromUser = (user = ADMIN_USER) => {
 
 const adminApolloServer = serverFromUser();
 export const queryAsAdmin = (request: any) => adminApolloServer.executeOperation(request);
+
+export const isSorted = (arr: []) => {
```

Comment:
⁉️

Reply:
Just an helper to check if an array is sorted.